python/amici/sbml_import.py

```diff
@@ -86,9 +86,10 @@ def __init__(
 
         Arguments:
 
-            sbml_source:  Either a path to SBML file where the model is specified.
-            Or a model string as created by  sbml.sbmlWriter().writeSBMLToString().
-            @type string
+            sbml_source: Either a path to SBML file where the model is
```

Comment:
on second thoughts, the argument ``from_file`` is not really necessary, because file path and sbml code are quite different in kind even though both are strings.

Reply:
Agreed. (Unless somebody saves his model with the name `<?xml`...) 
I didn't want to break anything unnecessarily though. Let's keep that for some future cleanup.